docs/usage/single.md

```diff
@@ -63,26 +63,7 @@ render((
 
 ### Custom labels for `enum` fields
 
-This library supports a custom [`enumNames`](https://github.com/rjsf-team/react-jsonschema-form/issues/57) property for `enum` fields, which, however is not JSON-Schema compliant (see below for a compliant approach).
-The `enumNames` property allows defining custom labels for each option of an `enum`:
-
-```jsx
-import validator from "@rjsf/validator-ajv6";
-
-const schema = {
-  type: "number",
-  enum: [1, 2, 3],
-  enumNames: ["one", "two", "three"]
-};
-
-render((
-  <Form schema={schema} validator={validator} />
-), document.getElementById("app"));
-```
-
```

Comment:
Either restore this, or maybe keep it deleted and instead add a note about `enumNames` being deprecated only